test/util/nightwatch-commands/fill.js

```diff
@@ -2,10 +2,15 @@
  * Clears the current value and if |value| is specified,
  *  enters the |value| in.
  */
-exports.command = function fill(selector, value) {
+exports.command = function fill(selector, value, callback) {
   this.clearValue(selector);
   if (typeof value !== 'undefined') {
-    this.setValue(selector, value);
+    // When callback is passed as undefined, setValue assumes selector is the locating strategy
```

Comment:
The comment is a little cryptic--was looking for an example of what sort of callback is passed in here in order to suggest something else, but I don't see it used anywhere.